README.md

```diff
@@ -395,6 +387,8 @@ After a few seconds (depending on block time), you see the receipt related to th
 
 ![Contract deployment transaction receipt](static/npm-deploy.png)
 
+> **Note:**: It will required to execute this command twice, first to pre-funded your sender account with enough tokens, then another one to send the actual deploy contract transaction.
```

Comment:
Some typos: 
"It will required" --> You will be required / You are required / ...
"first to pre-funded" --> First to fund

Reply:
Fixed